test/utils/jwt/src/proposals.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import fs from "fs";
+import * as Keys from "./keys";
+
+export const issuer = "http://Demo-jwt-issuer";
+
+export const issuerProposal = async (
+  host: string,
+  proposalsPath: string,
+): Promise<void> => {
+  // jwt issuer proposal documentation https://microsoft.github.io/CCF/main/build_apps/auth/jwt.html
+  const jwtIssuerProposal = {
+    actions: [
+      {
+        name: "set_jwt_issuer",
+        args: {
+          issuer: issuer,
+          key_filter: "all",
+          auto_refresh: false,
+        },
+      },
+      {
+        name: "set_jwt_public_signing_keys",
```

Comment:
How do we deal with IDP key rotation? Will we require new proposals when keys change?

Reply:
We can instruct CCF to refresh by means of auto_refresh: true in the proposal.